docs/design/flow-traffic-mirroring/flow-traffic-mirroring.md

```diff
@@ -0,0 +1,172 @@
+# Flow traffic mirroring
+
+## Idea
+There must be an ability to mirror traffic on a given flow and send it to a specific switch+port+vlan.
+
+## Model
+FlowPath has a list of FlowMirrorPoints, which will contain MirrorGroupId for multicasting on the switch. 
+FlowMirrorPoint itself has a list of MirrorFlowPaths. MirrorFlowPaths contains the endpoint 
+where the traffic should be mirrored and other necessary information to build the path.
+
+![DB model](./model.png "DB model")
+
+If the receiver point is on the mirroring source switch, then no segments will be built for such a MirrorFlowPath, 
+and we will work with it like the paths in one switch flow.
+
+## API
+* Create mirror point:
+  
+  `PUT /flows/{flow_id}/mirror`
+  
+  payload:
+  ```
+  {
+      "mirror_point_id": string,
+      "flow_id": string,
+      "mirror_direction": string [FORWARD|REVERSE],
+      "mirror_point_switch_id": string,
+      "receiver_point": {
+         "switch_id": string,
+         "port": int,
+         "inner_vlan": int,
+         "outer_vlan": int
+      }
+  }
+  ```
+
+* Delete mirror point:
+
+  `DELETE /flows/{flow_id}/mirror/{mirror_point_id}`
+
+
+* Get mirror points:
+
+  `GET /flows/{flow_id}/mirror`
```

Comment:
Is it missing "mirror_point_id"? Or it should be named "list mirror points"?

Reply:
Yea, it should be named "list of mirror points"